src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,41 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
```

Comment:
My instinct is that OntologyTerm it should go either in alleleAnnotations.avdl or perhaps in a separate file.